webapp/tests/utils/formatting_at_mentions.test.jsx

```diff
@@ -51,16 +51,16 @@ describe('TextFormatting.AtMentions', function() {
     });
 
     it('Implied at mentions', function() {
-        // PLT-4454 Assume users exist for things that look like at mentions until we support the new mention syntax
+        // implied at mention is presented as is and non-clickable
```

Comment:
Can you remove these tests down to the "Not at mentions" section? Between this PR and one of mine from a few days ago, "implied at mentions" won't exist any more